pages/landing.tsx

```diff
@@ -0,0 +1,82 @@
+import styled from '@emotion/styled';
+import Header from '@src/components/common/Header';
+import { useRouter } from 'next/router';
+import React, { useState, useEffect } from 'react';
+import Image from 'next/image';
+import planetImage from '@assets/planet_main.svg';
+import IntroduceSentence from '@src/components/common/IntroduceSentence';
+
+function landing() {
+  const router = useRouter();
+  const messages: string[] = [
+    '나만의 우주를 만들어보세요.\n행성이 감정을 담아둘거예요',
+    '함께 느꼈던 감정을 행성에 기록할 친구들을\nYouniverse로 초대할 수 있어요.',
+  ];
+  const [message, setMessage] = useState(messages[0]);
+
+  useEffect(() => {
+    let timer = setInterval(() => {
+      switch (message) {
+        case messages[0]:
+          setMessage(messages[1]);
+          break;
+        case messages[1]:
+          router.push('/onboarding');
+          break;
+      }
+    }, 1500);
+    return () => clearTimeout(timer);
+  }, [message]);
+
+  return (
+    <>
+      <Header />
+      <Styled.Main>
+        <Styled.TextWrapper>
+          <IntroduceSentence />
+          <p>{message}</p>
+        </Styled.TextWrapper>
+        <Styled.PlanetWrapper>
+          <Image src={planetImage} alt="planet image" />
+        </Styled.PlanetWrapper>
+      </Styled.Main>
+    </>
+  );
+}
+
+export default landing;
+
+const Styled = {
+  Main: styled.section`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  `,
```

Comment:
Main 부분은 main테그 쓰는 게 더 좋을 것 같아요!
그리고 저 부분 반복적으로 자주 나오는 스타일이라 lib/style/mixin.ts에 `flexColumnCenter`로 으진이가 정의해뒀는데 가져다 써도 좋을 것 같아요 
```suggestion
  Main: styled.main`
    ${flexColumnCenter}
  `,
```

Reply:
5fb3975a826011ccd7b181dd8eda7563a7ab9059